index.html

```diff
@@ -33,56 +35,38 @@ <h2 class="card-title">I’m a software developer</h2>
                 project you need coded, don’t hestiate to contact me.
               </p>
             </div>
-          <div>
-            <ul class="element">
-              <li>
-                <a href="#">
-                  <i class="bi bi-twitter text-white"></i>
-                </a>
-              </li>
-              <li>
-                <a href="#">
-                  <i class="bi bi-linkedin"></i>
-                </a>
-              </li>
-              <li>
-                <a href="#">
-                  <i class="bi bi-instagram"></i>
-                </a>
-              </li>
-              <li>
-                <a href="#">
-                  <i class="bi bi-github"></i>
-                </a>
-              </li>
-            </ul>
-          </div>
-        </section>
-      <section class="work">
-            <div class="row">
-            <div class="recent-work">
-                <h1 class="work-heading">My Recent Works</h1>
+            <div class="social-media">
+              <ul class="element">
+                  <li><a href=""><img src="images/1.png" alt="github"></a></li>
+                  <li><a href=""><img src="images/2.png" alt="instagram"></a></li>
+                  <li><a href=""><img src="images/3.png" alt="em"></a></li>
+                  <li><a href=""><img src="images/4.png" alt="twitor"></a></li>
+                  <li><a href=""><img src="images/5.png" alt="angellist"></a></li>
+              </ul>
             </div>
+          </div>  
+        </section>
+        <section class="work">
             <div class="row">
-                <div class="line">
+              <div class="recent-work">
+                  <h1 class="work-heading">My Recent Works</h1>
+              </div>
+              <div class="line-1">
                 <hr class="text-white" />
-                </div>
-            </div>
+              </div>
             </div>
             <div class="row-work">
-            <div class="work-card">
-                <img src="images/group.png" class="img-fluid" alt="#" />
-            </div>  
-                <div class="work-image">
-                </div>
+              <div class="work-card">
+                <img class="img-fluid" src="images/group.png" alt="#" />
+              </div>  
                 <h1 class="work-heading2">Multi-Post Stories</h1>
                 <p class="work-card-text">
-                A daily selection of privately personalized reads; no accounts or
```

Comment:
- [ ] Please fix the spacing of these elements. When you work on the desktop version, you should also keep the mobile version intact.
- [ ] Also please make sure there is no horizontal scrolling for different widths of the viewport. 
 
<p align="center">&nbsp;
 
| Mobile Version |   Desktop Version |
| ---------- | ----------- |
 | ![Screen Shot 2022-07-08 at 01 01 37](https://user-images.githubusercontent.com/12524453/177879208-280b834f-7df9-42b8-abe1-273781b54199.png) | ![FireShot Capture 154 - Portfolio - localhost](https://user-images.githubusercontent.com/12524453/177879772-b043694f-313d-41b7-a861-1125c4740ccf.png) |
</p>